src/makeCalculator.js

```diff
@@ -33,11 +33,63 @@
  *
  * calculator.result === -28
  *
- *
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  const add = num => {
+    result += num;
+
+    return this;
+  };
```

Comment:
This functions should work with this, so you need to add to **this**, same with other cases

Reply:
Lack of sleep affected my work, fixed the code to the proper style